pkg/pod/pod.go

```diff
@@ -283,6 +289,55 @@ func (builder *Builder) WaitUntilRunning(timeout time.Duration) error {
 	return builder.WaitUntilInStatus(corev1.PodRunning, timeout)
 }
 
+// WaitUntilHealthy waits for the duration of the defined timeout or until the pod is healthy.
+// An healthy pod is in running phase and optionally in ready condition.
+//
+// timeout is the duration to wait for the pod to be healthy
+// includeSucceeded when true, implies that pod in succeeded phase is running.
+// checkReadiness when true, to also checks that the podCondition is ready.
+// ignoreFailedPods when true, to Ignore failed pod with restart policy set to never.
+func (builder *Builder) WaitUntilHealthy(timeout time.Duration, includeSucceeded, checkReadiness,
```

Comment:
Please re-name arguments to:
`timeout time.Duration, includeSucceeded, skipReadinessCheck, ignoreRestartPolicyNever bool`

Reply:
fixed